src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/ParseRequestServiceTest.java

```diff
@@ -77,6 +75,14 @@ void shouldReturn400IfRoleTypeIsNotCaseForGetRoleAssignmentByActorIdAndCaseId()
         );
     }
 
+    @Test
+    void shouldReturn400IfRoleTypeIsEmptyforGetRoleAssignmentByActorIdAndCaseId() {
```

Comment:
Method name is too long. Could you please rename it?

Reply:
Done. Changed to getRoleAssignment_emptyCaseIdThrows400.